docker-compose.yml

```diff
@@ -201,7 +201,7 @@ services:
 
   # webKnossos-connect
   webknossos-connect:
-    image: scalableminds/webknossos-connect:master__190
+    image: scalableminds/webknossos-connect:master__205
```

Comment:
I just squeezed this in here, this is the upgrade to include the nicer error message.

Reply:
Nice, thank you!